mokapot/mokapot.py

```diff
@@ -82,9 +96,30 @@ def main():
                 dataset.add_proteins(proteins)
 
     # Define a model:
+    model = None
     if config.load_models:
         model = [load_model(model_file) for model_file in config.load_models]
-    else:
+    elif enabled_plugins:
+        plugin_models = {}
+        for plugin_name, plugin in enabled_plugins.items():
+            model = plugin.get_model(config)
+            if model is not None:
+                logging.debug(f"Loaded model for {plugin_name}")
+                plugin_models[plugin_name] = model
+
+        if len(plugin_models) == 0:
```

Comment:
**nitpik:** `if not plugin_models:` is more pythonic.

Reply:
but explicit is better than implicit!! (I am just kidding, I understand what you mean)